src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseAnalyzers.cs

```diff
@@ -23,6 +23,37 @@ public GivenThatWeWantToUseAnalyzers(ITestOutputHelper log) : base(log)
         {
         }
 
+        [Theory]
+        [InlineData("WebApp", "EnableRequestDelegateGenerator", false)]
+        [InlineData("WebApp", "EnableRequestDelegateGenerator", true)]
+        public void It_resolves_requestdelegategenerator_correctly(string testAssetName, string property, bool isEnabled)
+        {
+            var asset = _testAssetsManager
+                .CopyTestAsset(testAssetName, identifier: "C#")
+                .WithSource()
+                .WithProjectChanges(project =>
+                {
+                    var ns = project.Root.Name.Namespace;
+                    project.Root.Add(new XElement(ns + "PropertyGroup", new XElement(property, isEnabled)));
+                });
+
+            var command = new GetValuesCommand(
+                Log,
+                asset.Path,
+                ToolsetInfo.CurrentTargetFramework,
+                "Analyzer",
+                GetValuesCommand.ValueType.Item);
+
+            command
+                .WithWorkingDirectory(asset.Path)
+                .Execute("/bl")
```

Comment:
Try not to check in "/bl" arguments to test commands.  That just takes up space when tests run in CI, as there's no way to access the binlog.  For local test debugging you can temporarily add the parameter.